src/ApiService/ApiService/Functions/QueueFileChanges.cs

```diff
@@ -58,13 +58,7 @@ public class QueueFileChanges {
         var storageAccount = new ResourceIdentifier(topicElement.GetString()!);
 
         try {
-            // Setting isLastRetryAttempt to false will rethrow any exceptions
-            // With the intention that the azure functions runtime will handle requeing
-            // the message for us. The difference is for the poison queue, we're handling the
-            // requeuing ourselves because azure functions doesn't support retry policies
-            // for queue based functions.
-
-            var result = await FileAdded(storageAccount, fileChangeEvent, isLastRetryAttempt: false);
+            var result = await FileAdded(storageAccount, fileChangeEvent);
             if (!result.IsOk && result.ErrorV.Code == ErrorCode.ADO_WORKITEM_PROCESSING_DISABLED) {
```

Comment:
We're expecting FileAdded not to throw anymore so you need to update this if condition.

ADO_WORKITEM_PROCESSING_DISABLED -> Requeue 1 day
Other Errors -> Requeue with default visibility timeout

Reply:
Also I found a bug unrelated to your change but when we receive `ADO_WORKITEM_PROCESSING_DISABLED` and we requeue for a day, we still increment the retry count, which we shouldn't be doing since the work item creation should be 'paused'.
I'll make an issue for that.